### PR TITLE
fixes for attempt_submitted and attempt_reviewed events

### PR DESCRIPTION
**Pull request by davidpesce**
**Description**
- Moodle changed how they were tracking completion of quizzes.

**Related Issues**
- #129 

**PR Type**
- Fix for attempt_submitted and attempt_reviewed events

### CODE REVIEW

**Comment by davidpesce** [comment]
Don't merge this. It doesn't work (yet).

**Comment by davidpesce** [comment]
OK, ready for review! I tested this and it fixes the issue, but an extra set of eyes would be good.

File: lib/translator/src/Controller.php [hunk 1]
```diff
@@ -51,6 +51,7 @@ class Controller extends PhpObj {
         '\mod_quiz\event\attempt_preview_started' => 'AttemptStarted',
         '\mod_quiz\event\attempt_reviewed' => ['AttemptReviewed', 'QuestionSubmitted'],
         '\mod_quiz\event\attempt_viewed' => 'ModuleViewed',
+        '\mod_quiz\event\attempt_submitted' => ['AttemptSubmitted', 'QuestionSubmitted'],
```
Comment (ryasmi) [inline lib/translator/src/Controller.php:54]: Should Question Submitted be here?

**Comment by ryasmi** [comment]
Just that one comment, this seems okay @davidpesce

File: lib/translator/src/Controller.php (hunk 1 above)
Comment (davidpesce) [reply lib/translator/src/Controller.php:54]: I'm not sure. I made it as close to the attempt_reviewed as possible. I thought it was what was sending the statement about the individual questions.

**Pull request merged**